participant-datastore/enroll-mgmt-module/enroll-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/util/ErrorResponseUtil.java

```diff
@@ -110,8 +110,7 @@
     ERROR_REQUIRED("StudyId is required"),
     STUDYID_NOT_EXIST("StudyId does not exist"),
     TOKEN_ALREADY_USE("Token already in use"),
-    INVALID_TOKEN("Invalid Token"),
-    UNKNOWN_TOKEN("Unknown token"),
+    INVALID_TOKEN("Sorry, this token is invalid. Please enter a valid token to continue."),
     UNAUTHORIZED_CLIENT("Unauthorized client"),
     UNAUTHORIZED("Unauthorized"),
     INTERNAL_SERER_ERROR("Internal Serer Error"),
```

Comment:
Please update file copyright to `2020-2021`
While in this file, please fix misspelling of `server` at least in the text, if not in the name and uses also.

Reply:
Fixed comment.